dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceCodeRepoInterface.java

```diff
@@ -518,15 +518,11 @@ public WorkflowVersion combineVersionAndSourcefile(SourceFile sourceFile, Workfl
         // If source file is found and valid then add it
         if (sourceFile != null && sourceFile.getContent() != null) {
             version.getSourceFiles().add(sourceFile);
-            LOG.debug("Combining version " + version.getName() + " and source file " + sourceFile.getPath());
```

Comment:
FYI, I wouldn't remove these messages. I would just lower their visibility to, for example trace. But this could be useful if we get a reoccurrence.